### DISCUSSION TITLE
Popover stating sub comonents need a key.

### DISCUSSION THREAD

**Discussion by chiefpansancolt** [tailwindlabs/headlessui#491] [Help]
Does anyone have an example showing how to get rid of the Key console error when using popover for example with react? As a new react user still trying to understand its usage. Thanks!

**Comment by RobinMalfait** [comment]
Hey! Thank you for your question!
Much appreciated! 🙏

If you are using an array to render some links, then you have to provide keys. This is not a Headless UI / Popover issue, but a general React "issue". Checkout this link for more information on the topic: https://reactjs.org/docs/lists-and-keys.html
